lib/commands/ls.js

```diff
@@ -23,5 +23,6 @@ class LsCommand extends Command {
 
 LsCommand.description = 'View running ghost processes';
 LsCommand.global = true;
+LsCommand.aliases = ['status'];
```

Comment:
Based on how Ghost-CLI commands are loaded/invoked, adding aliases here won't work unfortunately.
The alias should instead be added here: https://github.com/TryGhost/Ghost-CLI/blob/master/lib/bootstrap.js#L13 - it's not the greatest solution for aliases but it's the only one that keeps the CLI from having to require all of the command files before running the one it needs.